altair/utils/schemapi.py

```diff
@@ -32,6 +32,7 @@
 import jsonschema.exceptions
 import jsonschema.validators
 from packaging.version import Version
+import narwhals as nw
```

Comment:
perhaps use `narwhals.stable.v1`, as that's what's used in the rest of the codebase?

Reply:
> perhaps use `narwhals.stable.v1`, as that's what's used in the rest of the codebase?

Thanks @MarcoGorelli, will give this a try later.

Should I be expected any behavioural changes from `narwhals.from_native`?

Reply:
at the moment they should be identical, just trying to future-proof things 😎

Reply:
Makes sense, cheers